### ISSUE TITLE
Add Media Modal Dialog Needs Tab Panel Controls

### ISSUE THREAD

**Issue by avaaec** [pressbooks/pressbooks#1782]
### Steps to Reproduce

1. Go to the Add Chapter page in the admin interface for a book
2. Navigate to the "Add Media" button directly above the TinyMCE application
3. Explore the resulting modal dialog and notice that the "links" on the left hand side of the dialog behave like tab controls, changing the rest of the content without taking the user to a new page.
4. Notice that there is a second layer of controls that behave like this inside of the first layer of tabs. This second layer of tabs is located at the top of the dialog. Under the "Add media" tab the second layer of tabs are called "Upload Media" and "Media Library". These "links" also behave like tabs and should be implemented as such.

**Expected behavior:** 
It is expected that the application structure of this dialog is implemented in an accessible way so that it is a seamless and intuitive experience for all people regardless of how they use a computer. Personally I think the easiest way to accomplish this on this page is to implement the current "links" as an aria tab panel: https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel.
**Actual behavior:** 
Right now the dialog appears to a non-visual user as a bunch of links that go nowhere. Without the structure of a tab panel, non-visual users don't know that the content of the dialog changes when they activate one of the "links" that act as a tab control. This is extremely confusing and frustrating for a non-visual user and is an almost unusable experience.

**Comment by SteelWagstaff** [comment]
The add media feature/tool is inherited from core WordPress and should probably be discussed upstream. I believe that several accessibility improvements were made to this dialog several years ago with WP 3.6 (https://make.wordpress.org/accessibility/2013/03/28/add-media-panel-and-wordpress-3-6-a-simple-solution/), but I haven't followed more recent conversations. To look into further.

**Comment by avaaec** [comment]
This is a UW ATS priority 2 issue

**Comment by SteelWagstaff** [comment]
@greatislander I believe this is also an upstream WP issue. Do you know any more about this that could help with the concerns raised in the initial ticket?

**Comment by greatislander** [comment]
This was fixed in WordPress 5.3; once Pressbooks updates to 5.3 this will be resolved. https://core.trac.wordpress.org/ticket/47149

**Comment by connermwallace** [comment]
University of Washington Accessibility Technology Services has inspected this issue and we have verified that it has been fixed. Version: 5.3.2.
